src/App.js

```diff
@@ -4,15 +4,58 @@ import Header from './Header.js';
 import Main from './Main.js';
 import Footer from './Footer.js';
 import './App.css';
+import Modal from 'react-bootstrap/Modal';
 
 class App extends React.Component {
 
+  //create constructor function
+  constructor(props){
+    super(props);
+    //how do we pass state to another component? through props
+    this.state = {
+      hornedbeast: '',
```

Comment:
don't forget to follow the proper naming convention here so that **hornedbeast** will be **hornedBeast**